public class ArrayExceptionExample{
    public static void main(String[]args){
        int[]numbers={1,2,3,4,5};
        try{
            System.out.println(numbers[9]);
        }catch(ArrayIndexOutofBoundsException e){
            System.out.println("Error:Array index is out of bounds");

        }
        System.out.println("program continues after handling exception");
        }
    }
}  